Code/Tools/AssetProcessor/Platform/Linux/AssetProcessor_Traits_Linux.h

```diff
@@ -9,3 +9,4 @@
 
 #define ASSETPROCESSOR_TRAIT_LEGACY_RC_RELATIVE_PATH "rc"
 #define ASSETPROCESSOR_TRAIT_CASE_SENSITIVE_FILESYSTEM true
+#define ASSETPROCESSOR_TRAIT_ASSET_BUILDER_LOST_CONNECTION_RETRIES 5
```

Comment:
Sorry to be so nit-picky, but can you change these files so that they end with a newline `\n` character?